rero_mef/authorities/utils.py

```diff
@@ -469,6 +388,7 @@ def write_link_json(
         agency_pid = viaf_pid
         add_schema(json_dump, 'viaf')
         json_dump['pid'] = agency_pid
+        del(json_dump['viaf_pid'])
```

Comment:
why?  is viaf_pid has changed to pid?

Reply:
Because the `pid`of a record is called `pid`